<launch>
  <node name="odrive" pkg="bumble" type="odrive" required="true" output="screen">
    <param name="max_speed" value="2.0"/>
  </node>
</launch>